comps/cores/proto/api_protocol.py

```diff
@@ -349,6 +349,7 @@ class ChatCompletionResponse(BaseModel):
     object: str = "chat.completion"
     created: int = Field(default_factory=lambda: int(time.time()))
     model: str
+    modalities: List[Literal["text", "audio"]] = Field(default=["text"])
```

Comment:
I think this should be in `ChatCompletionRequest` instead of `ChatCompletionResponse`, because it's the incoming request that is telling MMQnA if they want to get the response as text or as text and audio.